src/common/utils/snackbar.ts

```diff
@@ -0,0 +1,9 @@
+import { APIResponse } from "@types";
+
+export function errMessage(err: APIResponse<object>): string {
+  if (err?.message) {
+    return err?.message;
```

Comment:
```suggestion
    return err.message;
```

langsung, karena property .message sudah di cek di if block